modules/swagger-models/pom.xml

```diff
@@ -104,6 +104,11 @@
             <artifactId>mockito-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>3.5</version>
+        </dependency>
```

Comment:
We try not to add deps if not strictly necessary, use instead indexOf / charAt directly in code